src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
 // write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const headers = document.querySelectorAll('th');
+  const tableBody = document.querySelector('tbody');
+
+  const rows = Array.from(tableBody.querySelectorAll('tr'));
```

Comment:
Recommendation: Consider using more descriptive variable names. Variable names like 'headers' and 'rows' are quite generic. More descriptive names would make the code easier to read and understand.